test/video.js

```diff
@@ -11,6 +11,14 @@ tape('can downsample a video for a smaller filesize', (test) => {
   })
 })
 
+tape('can convert a video to webm', (test) => {
+  diff.video(test, {
+    input: 'videos/short.mp4',
+    expect: 'videos/short.webm',
+    options: {}
```

Comment:
I just noticed, should this actually be

```
options: {
  format: 'webm'
}
```

?

Reply:
I added the option.

Reply:
The bad thing is that the test was successful without the option...

That means that the test is not testing correctly...

Reply:
Good point. We should probably have a way to assert on some of the expected video metadata.